### DISCUSSION TITLE
ASA mod load order

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#572] [Suggestions]
Hey Max,

Please can we have the ability to change mod load order on mods page for ASA, same as ASE. 

Some mods are now required to be in certain places on the mod list - example https://www.curseforge.com/ark-survival-ascended/mods/tg-stacking-mod-10000-90

Thank you,
Gem

**Comment by mvaessen** [comment]
Hi Gem!

Thanks for your suggestion! This will be available in the upcoming update
